lib/active_storage/service/cloudinary_service.rb

```diff
@@ -81,7 +81,9 @@ def url_for_direct_upload(key, **options)
         # through direct upload (client side js), filename is missing, and that leads to inconsistent/broken URLs.
         # To avoid that, we explicitly pass file format in options.
         options[:format] = ext_for_file(key) if options[:resource_type] == "raw"
+        context = options.delete(:context)
         options[:context] = {active_storage_key: key}
+        options[:context].reverse_merge!(context) if context.respond_to?(:merge)
```

Comment:
I opted for the silent ignore here, but could also see a use case for removing the `if` clause and letting it fail if `reverse_merge!` isn't supported

Checked for `merge` because a `reverse_merge!` just calls `merge`